media/starboard/BUILD.gn

```diff
@@ -72,6 +72,12 @@ source_set("starboard") {
       "sbplayer_interface.h",
       "sbplayer_set_bounds_helper.cc",
       "sbplayer_set_bounds_helper.h",
+      "starboard_cdm.cc",
```

Comment:
Let's double check if all places using `COBALT_MEDIA_ENABLE_ENCRYPTED_PLAYBACKS` are addressed., remove the macro from this file if they are.